src/model-summary/index.jsx

```diff
@@ -6,10 +6,9 @@ import { connect } from 'react-redux';
 import { Snippet } from '../';
 
 const getValue = (value, format, model) => {
-  if (!value) {
-    return '-';
-  }
-  return format ? format(value, model) : value;
+  return format
+    ? format(value, model) || '-'
+    : value || '-';
```

Comment:
This feels very #jode-y. It's doing a lot of things in a single statement - specifically conditional formatting *and* default value-ing. I'd generally try to do one concept per "line".

```js
value = format ? format(value, model) : value;
return value || '-';
```

Reply:
FYI - You're welcome to say "no" and I'll merge anyway.

Reply:
you should stand for what you believe.